packages/dashboard/pages/resolve.tsx

```diff
@@ -17,7 +17,15 @@ import {
 
 import { FormEvent, useState } from 'react';
 
-const Resolver: NextPage = () => {
+export async function getServerSideProps(context: any) {
+  const res = await fetch(`http://${context.req.headers.host}/api/1.0`);
```

Comment:
same comment as before

Reply:
@OR13 correct me if I am wrong, but are saying we should import the uiConfigs into the getServerSideProps vs fetching from an end-point?

cc @acarnagey

Reply:
@ipbyrne I think that is correct no api call but import config actually

> Note: You can import modules in top-level scope for use in getServerSideProps. Imports used in getServerSideProps will not be bundled for the client-side.

This means you can write server-side code directly in getServerSideProps. This includes reading from the filesystem or a database.

Reply:
Copy that! Updated PR accordingly! @OR13 and @acarnagey .